interfaces/builtin/custom_device.go

```diff
@@ -43,7 +43,7 @@ const customDeviceBaseDeclarationSlots = `
     allow-installation: false
     allow-connection:
       plug-attributes:
-        content: $SLOT(custom-device)
+        custom-device: $SLOT(custom-device)
```

Comment:
this isn't too material probably, because anything using it needs its own declaration as the slots of the interface are superprivileged